cmd/testscript/processor/aggregator.go

```diff
@@ -0,0 +1,30 @@
+package processor
+
+import (
+	"fmt"
+	"sort"
+)
+
+var percentilesToCalculate = []int{50, 75, 80, 90, 95}
+
+type Aggregator struct {
+	runs []*Run
+}
+
+func NewAggregator(runs []*Run) *Aggregator {
+	sort.Slice(runs, func(i, j int) bool {
+		return runs[i].TimeElapsed < runs[j].TimeElapsed
+	})
+	return &Aggregator{runs: runs}
+}
+
+func (a *Aggregator) Percentile(percentile int) string {
+	index := len(a.runs)*percentile/100 - 1
+	return a.runs[index].TimeElapsed.String()
```

Comment:
We need to test if `index >= 0` to prevent panics when `len(a.runs)*percentile < 100`.